pom.xml

```diff
@@ -500,7 +500,7 @@
 			<dependency>
 				<groupId>org.dbunit</groupId>
 				<artifactId>dbunit</artifactId>
-				<version>2.5.4</version>
+				<version>2.5.4 </version>
```

Comment:
@georgejunior is this an upgrade or just whitespace?

Reply:
@samuelmale i first did the required changes (upgrade) then as i was making some changes, i just added the whitespace

Reply:
Where is the upgrade, you mean it was already merged into core? If not, I don't any work done here @georgejunior .

Reply:
So are you going to make the required update? You can look at this https://mvnrepository.com/artifact/org.dbunit/dbunit to see the latest update